modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z_{context}"]
+= {ibm-z-title} only: Manually adding {ibm-z-title} agents
+
+After creating the PXE assets you can add {ibm-z-name} agents. Depending on your {ibm-z-name} environment you can choose from the following:
+
+* Adding {ibm-z-name} agents with z/VM
+* Adding {ibm-z-name} agents with {op-system-base} KVM
+
+.Procedure
```

Comment:
I think that this should probably be revised into two separate modules. There are two procedures here. I like this organization really, but I don't think it complies with our one-procedure-per-assembly rule: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures " In most cases, create your procedures as individual modules. . ."